lib/glueby/contract/active_record/timestamp.rb

```diff
@@ -5,11 +5,12 @@ class Timestamp < ::ActiveRecord::Base
         include Glueby::Contract::Timestamp::Util
         enum status: { init: 0, unconfirmed: 1, confirmed: 2 }
 
+        # @param [String] wallet_id hex string which represents identifier of a wallet
         # @param [String] content Data to be hashed and stored in blockchain.
         # @param [String] prefix prefix of op_return data
-        def initialize(content:, prefix: '')
+        def initialize(wallet_id:, content:, prefix: '')
```

Comment:
Since the argument of `Glueby::Contract::Timestamp` acccept a wallet object, isn't it okay to use a wallet object as well?

Reply:
In many use-case, we don't need to load wallet itself in controller action.

```
class TimestampController extend ApplicationController
  def create
    timestamp = Timestamp.new(create_params)
    timestamp.save
    ....
  end
  
  def create_params
    params.fetch(:timestamp, {}).permit(:wallet_id, :content)
  end
end
```

Reply:
ok.